packages/server/src/sdk/app/backups/imports.ts

```diff
@@ -25,9 +28,29 @@ type TemplateType = {
   key?: string
 }
 
+function generateAttachmentFindParams(
+  attachmentCols: string[],
+  bookmark: null | string
+) {
+  const params: CouchFindOptions = {
+    selector: {
+      _id: {
+        $regex: `^${TABLE_ROW_PREFIX}`,
+      },
+    },
+    limit: FIND_LIMIT,
+  }
+  attachmentCols.forEach(col => (params.selector[col] = { $exists: true }))
+  if (bookmark) {
+    params.bookmark = bookmark
+  }
+  return params
+}
+
 async function updateAttachmentColumns(prodAppId: string, db: Database) {
```

Comment:
Can we `export` this function so we can add a unit test for it? 

A db aware test that can:
- Create table
- Add attachment rows 
- Call `updateAttachmentColumns`
- Read back rows and ensure urls is in correct format

Reply:
Preferably we could decouple the pagination from `updateAttachmentColumns`. In the ideal scenario the pagination would live in the rows sdk:

```ts

interface PageOpts {
   limit: number,
   bookmark: string
}

interface PageResponse<T> {
   bookmark: string,
   rows: T[] 
}

// specific function for creating `CouchFindOptions` for attachments search
// uses `getPage`
sdk.rows.getAttachmentRows(appId: string, opts: PageOpts): PageResponse<Row>

// generic pagination function
// defaults for `FIND_LIMIT` live here
sdk.rows.getPage(opts: CouchFindOptions): PageResponse<Row>
```

This is beneficial for a few resons:
- Enable us to unit test pagination in isolation
- Enable us to unit test attachment rows retrieval in isolation
- Allow us to supply an arbitrary page size to result in faster test execution
- Remove the concerns of pagination from the test for `updateAttachmentColumns`
- Encapsulate db logic in rows sdk and build up more re-usable functions